### PR TITLE
Fixes for Windows

### PR DESCRIPTION
**Pull request by davedoesdev**
Tests now pass and it compiles on Windows!

Question:

- Is `utime` still needed given `fs.utimes` exists?

### CODE REVIEW

**Comment by davedoesdev** [comment]
Appveyor doesn't seem to be picking up change to run on Node 8

**Comment by davedoesdev** [comment]
Travis and Appveyor both building successfully.

**Comment by davedoesdev** [comment]
Works on Node 8.1.3. Might work on earlier versions, not sure when https://github.com/libuv/libuv/pull/1323 went into Node.

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,10 @@ notifications:
 language: node_js
 dist: trusty
 node_js:
-    - "0.10.45"
-    - "7"
+    - "0.12"
```
Comment (baudehlo) [inline .travis.yml]: It's high time we removed support for this.

File: package.json [hunk 2]
```diff
@@ -13,21 +18,25 @@
   "engines": {
     "node": ">= v0.8.0"
```
Comment (baudehlo) [inline package.json]: Can you please bump this to >= v6. It's a shame we can't do platform specific versions given Windows now requires v8. I searched and the only solution is a failing install script.

File: .travis.yml (hunk 1 above)
Comment (baudehlo) [inline .travis.yml]: And by "this" I mean anything less than node v6 on travis.

**Comment by davedoesdev** [comment]
@baudehlo sure see latest commit

**Comment by davedoesdev** [comment]
@baudehlo is there anything else you need on this PR?

**Comment by baudehlo** [comment]
I don't think so. Just need some free time (baby, new house, etc).

**Comment by davedoesdev** [comment]
@baudehlo been there - take time for what's important.

**Comment by baudehlo** [comment]
Merged

**Comment by baudehlo** [comment]
BTW after merge Travis fails. Not sure why.

**Comment by davedoesdev** [comment]
Node 8 behaviour has changed for the -1 utime test. It sets it fine (and Node 6 and Python read the atime as -1). But for Node 8 atime in that case is NaN.

**Comment by davedoesdev** [comment]
@baudehlo how do `utime` and `utimeSync` provided by `fs-ext` differ from `fs.futimes` and `fs.futimesSync` don't?

**Comment by baudehlo** [comment]
I think those were just added after I released this.

**Comment by davedoesdev** [comment]
How would you feel about removing them or making them aliases?

**Comment by baudehlo** [comment]
Sounds good to me.

**Comment by davedoesdev** [comment]
#71

**Pull request merged**